common/src/main/java/com/discordsrv/common/feature/linking/LinkingRewardsModule.java

```diff
@@ -44,6 +44,22 @@ public LinkingRewardsModule(DiscordSRV discordSRV) {
     public void onPlayerConnected(PlayerConnectedEvent event) {
         discordSRV.profileManager().queryProfile(event.player().uniqueId())
                 .whenSuccessful(profile -> {
+                    Set<RewardsConfig.Reward> pendingRewards = new HashSet<>();
```

Comment:
We should run these in order so there's some way to account for link and unlink commands being ran at around the same time
```suggestion
                    Set<RewardsConfig.Reward> pendingRewards = new LinkedHashSet<>();
```